modular_skyrat/modules/cargo/code/packs.dm

```diff
@@ -107,6 +107,14 @@
 					/obj/item/weaponcell/medical/toxin)
 	crate_name = "medicell replacement crate"
 
+/datum/supply_pack/medical/syringe_gun
+	name = "Syringe Gun"
+	desc = "A replacement medical syringe gun"
```

Comment:
```suggestion
	desc = "A replacement medical syringe gun."
```

Reply:
I was trying to keep consistency with the other ones, but they really aren't super consistent.